app/views/groups/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="header">
+<%= link_to "Back", groups_path, class: "header-link" %>
+<h2 class="title">Transactions</h2>
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=Design-,You%20should%20follow%20these%20design%20guidelines%2C%20including,-%3A), Please make the back button a back arrow like this `<`, not the word `back` as you did to match the given design. and kindly change it for all views that have a back button on it.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://github.com/Swarzstein/ror-capstone/assets/68796232/4089c136-2688-41b1-9855-8cc5f25b4aff)  | ![image](https://github.com/Swarzstein/ror-capstone/assets/68796232/10f92a0a-0e41-44ca-9e2e-c9622595dcd0)  |